src/util.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {serveFile}
```

Comment:
is this file even used?

Reply:
Hmm.. I don't think so. Good catch. I think I started writing something and then abandoned it.